dokka-subprojects/plugin-base/src/test/kotlin/expectActuals/ExpectActualsTest.kt

```diff
@@ -732,4 +734,56 @@ class ExpectActualsTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Test
+    @OnlySymbols("New KDoc resolve")
+    @OptIn(ExperimentalDokkaApi::class)
+    fun `kdoc on expect-actual`() {
+        testInline(
+            """
+        /src/common/test.kt
+        /**
+         * @constructor Common Description
+         */
+        expect class A() {
+            /**
+             * Common Description
+             */
+            fun foo()
+            
+            /**
+             * Common Description
+             */
+            fun bar()
+        }
+        
+        /src/jvm/test.kt
+        actual class A actual constructor() {
+            actual fun foo() {}
+            
+            /**
+             * Platform Description
+             */
+            actual fun bar() {}
+        }
+        """.trimMargin(),
+            multiplatformConfiguration
+        ) {
+            documentablesTransformationStage = { module ->
+                val a = module.findClasslike("", "A") as DClass
+                val constructor = a.constructors.single()
+                val foo = a.functions.first { it.name == "foo" }
+                val bar = a.functions.first { it.name == "bar" }
+
+                assertEquals(2, constructor.documentation.size)
```

Comment:
It would be nice to check not just sizes, but some other content, like documentation override in case of `bar` on JVM, or source-sets, otherwise it's not clear what the test is really tests :)